src/legacy/core_plugins/vis_type_vega/public/vega_visualization.js

```diff
@@ -103,13 +103,14 @@ export const createVegaVisualization = ({ serviceSettings }) =>
           this._vegaView = null;
         }
 
-        const { filterManager } = npStart.plugins.data.query;
+        const { filterManager: queryfilter } = this.dataPlugin.query;
```

Comment:
We mostly don't use the term `queryfilter` anymore.
Lets stick to `filterManager`?

Reply:
np, done